libcontainerd/client_windows.go

```diff
@@ -328,6 +329,18 @@ func (clnt *client) createLinux(containerID string, checkpoint string, checkpoin
 		})
 	}
 
+	for _, option := range options {
+		if n, ok := option.(*NetworkEndpointsOption); ok {
+			configuration.EndpointList = n.Endpoints
+			configuration.AllowUnqualifiedDNSQuery = n.AllowUnqualifiedDNSQuery
+			if n.DNSSearchList != nil {
+				configuration.DNSSearchList = strings.Join(n.DNSSearchList, ",")
+			}
+			configuration.NetworkSharedContainerName = n.NetworkSharedContainerID
+			continue
```

Comment:
If there is more than one NetworkEndpointsOption (will not happen with the current caller), is the idea that this loop continues, and uses the last one? In that case, the continue is not needed here. If there is only ever expected to be one, then `continue` should be replaced with `break`.